kolibri/core/content/utils/channel_import.py

```diff
@@ -50,7 +50,10 @@
 models_not_to_overwrite = [LocalFile]
 
 models_to_exclude = [
-    apps.get_model(CONTENT_APP_NAME, "ChannelMetadata_included_languages")
+    apps.get_model(CONTENT_APP_NAME, "ChannelMetadata_included_languages"),
```

Comment:
Doesn't need to be fixed here, but this is making me realize we should probably be accounting for these `models_to_exclude` when we do the SQLAlchemy schema generation. No need to generate a schema for models we are not importing from or to.

Reply:
I'll look into this